pkgs/tools/filesystems/ssdfs-utils/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, libtool
+, libuuid
+, zlib
+}:
+
+stdenv.mkDerivation {
+  pname = "ssdfs-utils";
+  version = "4.24";
+
+  src = fetchFromGitHub {
+      owner = "dubeyko";
+      repo = "ssdfs-tools";
+      rev = "0a3ca85b454e56cf2422f33663639ad13fc792dd";
+      hash = "sha256-pzpwbvo0BR4fG6Cnxb9GdH/UZlJ6C5am7TULSLEzdvI=";
```

Comment:
In Nixpkgs we use two spaces for indentation of Nix code.

Reply:
Whoops, somehow the autoformatter didn't fire. :sweat_smile: